.github/workflows/knative-releasability.yaml

```diff
@@ -0,0 +1,138 @@
+# Copyright 2020 The Knative Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is automagically synced here from github.com/knative-sandbox/.github
+# repo by knobots: https://github.com/mattmoor/knobots and will be overwritten.
+
+name: 'Releasability'
+
+on:
+  schedule:
+    - cron: '0 1 * * 1-5' # 6am Pacific, weekdays.
+
+  workflow_dispatch:      # Manual trigger.
+    inputs:
+      releaseFamily:
+        description: 'Release? (vX.Y)'
+      slackChannel:
+        description: 'Slack Channel? (release-#)'
+
+jobs:
+  releasability:
+    name: Releasability
+    runs-on: 'ubuntu-latest'
+
+    env:
+      #########################################
+      #   Update this section each release.   #
+      RELEASE: 'v0.18'
```

Comment:
This seems inappropriate to include for this repo?

Reply:
Hmm, you are right.

cc @n3wscott any thoughts?

Reply:
hmmm we will have to update the update script to reject some things, it is not clear that ko wants any of the knative actions